test/functional/Jsr292/playlist.xml

```diff
@@ -102,6 +102,32 @@
 		</subsets>
 	</test>
 
+	<test>
+		<testCaseName>jsr292Test_jdk12</testCaseName>
+		<variations>
+				<variation>NoOptions</variation>
+				<variation>Mode195</variation>
+		</variations>
+		<command>$(JAVA_COMMAND) $(JVM_OPTIONS) \
+	--add-opens=java.base/java.lang=ALL-UNNAMED \
+	-Djava.security.policy=$(Q)$(TEST_RESROOT)$(D)java.policy$(Q) \
+	-cp $(Q)$(TEST_RESROOT)$(D)jsr292test.jar$(P)$(RESOURCES_DIR)$(P)$(TESTNG)$(P)$(LIB_DIR)$(D)asm-all.jar$(Q) \
+	org.testng.TestNG -d $(REPORTDIR) $(Q)$(TEST_RESROOT)$(D)testng.xml$(Q) \
+	-testnames jsr292Test_Java12up \
+	-groups $(TEST_GROUP) \
+	-excludegroups $(DEFAULT_EXCLUDE); \
+	$(TEST_STATUS)</command>
+		<levels>
+			<level>extended</level>
+		</levels>
+		<groups>
+			<group>functional</group>
+		</groups>
+		<subsets>
+			<subset>12+</subset>
+		</subsets>
```

Comment:
Could you please add impls? Thanks

```
		<impls>
			<impl>openj9</impl>
			<impl>ibm</impl>
		</impls>
```

Reply:
these changes should not be j9 specific.

Reply:
Ideally try running these tests against a hotspot build, have you been able to verify that?
If so and they run fine, then you can leave out the `<impls>` block, and impl specifiers, so that these tests will get run against all implementations?

Reply:
the NoOptions run passes with hotspot but the Mode195 test does not.

Reply:
So presumably a hotspot build chokes on the option -XX:RecreateClassfileOnload which [Mode195 contains](https://github.com/eclipse/openj9/blob/master/test/TestConfig/resources/modes.xml#L1979).

Reply:
makes sense. its added now

Reply:
In this case, should we break this test into two? one for NoOptions which is for all impls, and one for Mode195 which is openj9 and ibm specific?

Reply:
I was not going to suggest it, because it has implications for all other targets (as test team already discussed, and have deferred decision), we may want to consider other options than nearly duplicate test targets for the multi-impl test approach.

Reply:
but yes, it could broken it in 2.  

Eventually, we will need to do a pass of all playlists, when we do that, we may not want to break all the test targets into 2 to achieve multi-impl testing.  We may want a rule that says NoOptions gets run on all impls, or some back-end check for options that strips out modes that are not applicable.